library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java

```diff
@@ -170,8 +170,15 @@ private boolean copyableMessage() {
 
     private boolean threadableMessage() {
         return style.isEnableThread()
+                && viewModel.getChannel().getConfig().isReplies()
```

Comment:
`isReplies` is a pretty terrible name; what about `repliesEnabled` instead

Reply:
`isReplies` was created by _Generate_ _Getter_ and _Setter_ automatically.
Sure we can change it as `repliesEnabled`